server/appcap.go

```diff
@@ -91,13 +91,13 @@ func (s *IDPServer) addGrantAccessContext(handler http.HandlerFunc) http.Handler
 
 		who, err = s.lc.WhoIs(r.Context(), remoteAddr)
 		if err != nil {
-			http.Error(w, fmt.Sprintf("Error getting WhoIs: %v", err), http.StatusInternalServerError)
+			writeHTTPError(w, r, http.StatusInternalServerError, "server_error", fmt.Sprintf("Error getting WhoIs: %v", err))
```

Comment:
these error codes (e.g. `server_error`) look repetitive and somewhat consistent.
could you define some constants for them?